python_scripts/metrics_sol_02.py

```diff
@@ -91,3 +91,41 @@
 }
 scores = pd.DataFrame(scores)
 scores
+
+# %% [markdown] tags=["solution"]
+# In the Regression Metrics notebook we mentioned the concept of loss function,
+# which is the metric to be optimized when training a model. In the case of a
+# linear regression, the fitting process consists in minimizing the mean squared
```

Comment:
I find "linear regression" to be to vague here. I would instead refer to `LinearRegression` that is the scikit-learn estimator minimizing the MSE.